code/modules/atmospherics/machinery/components/unary_devices/outlet_injector.dm

```diff
@@ -17,6 +17,48 @@
 	///Rate of operation of the device
 	var/volume_rate = 50
 
+	///id of air sensor its connected to
+	var/chamber_id
+
+/obj/machinery/atmospherics/components/unary/outlet_injector/Initialize(mapload)
+	. = ..()
+	var/static/list/multitool_tips = list(
+		TOOL_MULTITOOL = list(
+			SCREENTIP_CONTEXT_LMB = "Log it to link later with an air sensor",
+		)
+	)
+	AddElement(/datum/element/contextual_screentip_tools, multitool_tips)
+
+/obj/machinery/atmospherics/components/unary/outlet_injector/examine(mob/user)
+	. = ..()
+	. += span_notice("You can link it with an air sensor using a Multitool")
+
+/obj/machinery/atmospherics/components/unary/outlet_injector/multitool_act(mob/living/user, obj/item/multitool/multi_tool)
+	. = ..()
+	if (!istype(multi_tool))
+		return .
+
+	balloon_alert(user, "saved in buffer")
+	multi_tool.buffer = src
+	return TRUE
+
+/obj/machinery/atmospherics/components/unary/outlet_injector/wrench_act(mob/living/user, obj/item/I)
+	. = ..()
+	if(.)
+		disconnect_chamber()
+
+///called when its either unwrenched or destroyed
+/obj/machinery/atmospherics/components/unary/outlet_injector/proc/disconnect_chamber()
+	if(chamber_id != null)
+		var/input_id = CHAMBER_INPUT_FROM_ID(chamber_id)
+		GLOB.objects_by_id_tag[input_id] = null
```

Comment:
Why bother setting to null when you `-=` right after?

Reply:
Just wanted to be sure but anyway ill remove it